deployment/secure_api_server.py

```
@@ -229,18 +237,22 @@
+                logger.info("Torch not available, using CPU")
 
             self.loaded = True
-            logger.info("✅ Secure model loaded successfully")
+            logger.info(
+                "Secure model loaded successfully"
+            )
 
         except Exception as e:
-            logger.error(f"❌ Failed to load secure model: {str(e)}. Falling back to stub mode.")
+            logger.error(
+                f"Failed to load secure model: {str(e)}. Falling back to stub mode."
```

Ruff (0.12.2)

Use explicit conversion flag

Replace with conversion flag

(RUF010)